engine/src/engine.lcb

```diff
@@ -406,4 +411,68 @@ begin
     MCEngineExecLogWithValues(Message, Arguments)
 end syntax
 
+/**
+Summary: 	Fetches the 'columnDelimiter' property from script context
+
+Example:
+	split tString by the column delimiter
+
+Description:
+Fetches the calling (script) handler's columnDelimiter property. If there is no
+script context, then it returns tab.
```

Comment:
Some general points for all the documentation blocks! Please:

* use inline code markup where applicable (e.g. for `columnDelimiter`) in this example
* make sure examples can be compiled and run if they're simply copy-and-pasted into a source file with no further alterations, and show what the result is likely to be. E.g.

        Examples:
            -- Split a string into a list using the column delimiter
            variable tItems
            put "alice\tbob\teve" into tItems
            split tItems by the column delimiter
            -- tItems is now a list: ["alice", "bob", "eve"]

* add cross-references to related documentation items, and appropriate tags

Also, in this description, "it" is unclear.  Consider something like:

> Fetch the calling (script) handler's `columnDelimiter` property, or horizontal tab (U+0009) if no script context is available.